CMakeLists.txt

```diff
@@ -5,24 +5,24 @@
 # lowest version supporting C_STANDARD property
 cmake_minimum_required (VERSION 3.1.3)
 
+# Set software information
+project(libsndfile VERSION 1.0.28.0)
+
```

Comment:
Why does that have the `.0` on the end. I have always used an `x.y.x` version number system for libsndfile.